pkg/auth/auth.go

```diff
@@ -26,6 +26,8 @@ const (
 	hostsKey              = "hosts"
 	localhost             = "github.localhost"
 	oauthToken            = "oauth_token"
+	// TenancyHost is the domain name of a tenancy GitHub instance.
+	tenancyHost = "ghe.com"
```

Comment:
Looking at this I asked myself _"Does VS Code really not try to fix the formatting on this one?"_ only to realize, no it doesn't.

Not the end of the world, but would rather the longer line with consistent formatting than shorter lines that are inconsistent. Not end of the world 🤷 

```suggestion
	tenancyHost           = "ghe.com" // TenancyHost is the domain name of a tenancy GitHub instance.
```

Reply:
Seems reasonable to me as it also bothered me 😅